WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController+QuickStart.swift

```diff
@@ -13,17 +13,18 @@ extension MySiteViewController {
             return
         }
 
-        siteMenuSpotlightIsShown = element == .siteMenu
+        switch element {
+        case .siteIcon, .siteTitle, .viewSite, .pages, .stats, .mediaScreen, .sharing:
```

Comment:
Turns out we also need to add .siteMenu to get it to scroll for the social sharing tour
```suggestion
        case .siteIcon, .siteTitle, .viewSite, .pages, .stats, .mediaScreen, .sharing, .siteMenu:
```